<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>449</width>
    <height>523</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="baseSize">
   <size>
    <width>614</width>
    <height>684</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Handwriting recognition</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../Static/A-0.png</normaloff>../Static/A-0.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>411</width>
      <height>471</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <property name="movable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Recognize</string>
     </attribute>
     <widget class="QLabel" name="ImgToShow">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>381</width>
        <height>181</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../../Data/Obrazki_do_uczenia/B/B-0.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="LabelOutputCheck">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>220</y>
        <width>381</width>
        <height>61</height>
       </rect>
      </property>
      <property name="statusTip">
       <string comment="1"/>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <property name="text">
       <string>I didn't check what it is sorry</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
      <property name="margin">
       <number>2</number>
      </property>
     </widget>
     <widget class="QPushButton" name="MLcheck">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>390</y>
        <width>381</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Check what is on the image</string>
      </property>
     </widget>
     <widget class="QPushButton" name="ChgIMGButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>290</y>
        <width>381</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Change your image</string>
      </property>
     </widget>
     <widget class="QPushButton" name="MLbasefolder_butt">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>340</y>
        <width>381</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Change neuron layer dir</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Teach neuron layers</string>
     </attribute>
     <widget class="QPushButton" name="TeachLayer">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>400</y>
        <width>391</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Teach </string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>391</width>
        <height>31</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
      <property name="text">
       <string>Your paths</string>
      </property>
      <property name="textFormat">
       <enum>Qt::RichText</enum>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="LabelStatistic">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>160</y>
        <width>391</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>6</pointsize>
       </font>
      </property>
      <property name="frameShape">
       <enum>QFrame::Panel</enum>
      </property>
      <property name="text">
       <string>No place were to save data</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="FolderVector">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>110</y>
        <width>391</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Chose traning folder</string>
      </property>
     </widget>
     <widget class="QPushButton" name="FolderStatistic">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>210</y>
        <width>391</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Chose folder to save data</string>
      </property>
     </widget>
     <widget class="QLabel" name="LabelPathVector">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>391</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>6</pointsize>
       </font>
      </property>
      <property name="frameShape">
       <enum>QFrame::Panel</enum>
      </property>
      <property name="text">
       <string>No path to traning vector</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>350</y>
        <width>211</width>
        <height>41</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
      <property name="text">
       <string>Amount of loops per click:</string>
      </property>
      <property name="textFormat">
       <enum>Qt::RichText</enum>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLineEdit" name="LoopPclick">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>350</y>
        <width>171</width>
        <height>41</height>
       </rect>
      </property>
      <property name="inputMask">
       <string/>
      </property>
      <property name="text">
       <string>500</string>
      </property>
      <property name="maxLength">
       <number>32767</number>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QProgressBar" name="progressBar">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>260</y>
        <width>391</width>
        <height>31</height>
       </rect>
      </property>
      <property name="value">
       <number>0</number>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>300</y>
        <width>211</width>
        <height>41</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
      <property name="text">
       <string>Amount data from vector:</string>
      </property>
      <property name="textFormat">
       <enum>Qt::RichText</enum>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLineEdit" name="LoopPclick_2">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>300</y>
        <width>171</width>
        <height>41</height>
       </rect>
      </property>
      <property name="inputMask">
       <string/>
      </property>
      <property name="text">
       <string>1</string>
      </property>
      <property name="maxLength">
       <number>32767</number>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>449</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionTest_AI">
   <property name="text">
    <string>Test AI</string>
   </property>
  </action>
  <action name="actionTeach_AI">
   <property name="text">
    <string>Teach AI</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
